pkgs/by-name/uv/uv/package.nix

```diff
@@ -65,8 +64,13 @@ python3Packages.buildPythonApplication rec {
 
   pythonImportsCheck = [ "uv" ];
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = [ "--version" ];
```

Comment:
Is this new? I've never seen that before.

Reply:
I don't know how new it is, but I discovered it this week.
It's pretty nice as it performs the check at build time so it is less likely to be forgotten.

Reply:
Is it better to do it at build time or put it in `passthru.tests.version` ? What are the pros and cons?
Just to be clear, I'm not blocking this PR, this is for my own personal information

Reply:
> It is also possible to use passthru.tests to test the version with [testVersion](https://nixos.org/manual/nixpkgs/unstable/#tester-testVersion), but since that is pretty trivial and recommended thing to do, we recommend using [versionCheckHook](https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook) for that...

See [here](https://nixos.org/manual/nixpkgs/unstable/#var-passthru-tests-packages) for the list of advantages.

Reply:
Super interesting, thank you !

Reply:
Yes, I will try to use this more and more from now on.